bot.py

```diff
@@ -481,7 +485,9 @@ async def vaccines(ctx, loc="United States"):
     if dat is not None:
         #Format the number with commas to make it easier to read
         tot = "{:,}".format(int(dat['total_vaccinations']))
-        msg = f"In {loc} as of {dat['date']}, there have been {tot} vaccinations, totalling {dat['total_vaccinations_per_hundred']}% of the population."
+        msg = f"In {loc} as of {dat['date']}, there have been {tot} \
+         vaccinations, totalling {dat['total_vaccinations_per_hundred']}% of \
+         the population."
```

Comment:
Has there been any testing in how this formats the printing of the strings? Python can be funky with how it prints -- interpreting the new line as a tab or something. No idea.

Reply:
Figured it out will put in next commit